shared/java/org/jetbrains/jwm/MouseCursor.java

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.jwm;
+
+import org.jetbrains.annotations.*;
+
+public enum MouseCursor {
+    ARROW  (0),
+    CROSS  (1),
+    HAND   (2),
+    HELP   (3),
+    IBEAM  (4),
+    UPARROW(5);
+
+    @ApiStatus.Internal public final int _id;
```

Comment:
Let’s use `_values` to transform enum <-> int, just for consistency with Key.java

Reply:
ID used to transform java enum to c++. 
There is no need to transform c++ to java => no values needed

Reply:
IBEAM.ordinal() will get you the integer

Reply:
👌